app/helpers/application_helper/toolbar_builder.rb

```diff
@@ -496,6 +496,10 @@ def build_toolbar_hide_button(id)
     return true if id == 'miq_request_edit' &&
                    %w(ServiceReconfigureRequest ServiceTemplateProvisionRequest).include?(@miq_request.try(:type))
 
+    # hide power management buttons for Openstack::InfraManager
+    return true if %w(host_standby host_shutdown host_reboot host_start host_stop host_reset).include?(id) &&
+                   @record.kind_of?(ManageIQ::Providers::Openstack::InfraManager)
```

Comment:
Not a fan of hardcoding provider specific logic into the UI...makes pluggability harder.  @blomquisg Thoughts here?